.github/workflows/smoke-test.yml

```diff
@@ -126,13 +126,24 @@ jobs:
               --driver ${{ matrix.driver }} \
               ${{ matrix.network_flag }} \
               ${{ env.LOG_ARGS }}
+      - name: Copy testdata for image build
+        run: |
+          echo ">>> Copying testdata to ./out"
```

Comment:
This log does not add any value, and it hides the actual command if the step command is folded. Please remove it.